stable/gke-node-termination-handler/values.yaml

```diff
@@ -24,3 +24,13 @@ resources:
   requests:
     cpu: 150m
     memory: 30Mi
+
+# args -- Command arguments. Usually you don't need to override them.
```

Comment:
Maybe remove helm-docs comment so it's not shown in README? I don't think anyone would ever use this.

Reply:
I think it's fine to leave them. I would rather over-document than under.